helm-elscreen.el

```diff
@@ -1,12 +1,12 @@
-;;; helm-elscreen.el --- Elscreen support -*- lexical-binding: t -*-
+;;; helm-elscreen.el --- Elscreen with helm interface -*- lexical-binding: t -*-
 
 ;; Copyright (C) 2012 ~ 2017 Thierry Volpiatto <thierry.volpiatto@gmail.com>
 
 ;; Author: Thierry Volpiatto <thierry.volpiatto@gmail.com>
 ;; Version: 1.0
 ;; Keywords: files, convenience
 ;; URL: https://github.com/emacs-helm/helm-elscreen
-;; Package-Requires: ((emacs "24"))
+;; Package-Requires: ((emacs "24") (helm "2.8.0") (elscreen "0") cl-lib)
```

Comment:
Here you should include the version for `cl-lib`, ie. `(cl-lib "0.5")`. Older versions of `package.el` can't parse the format you're using here, so it's best to always use the old format.